dqc_us_rules/dqc_us_0005.py

```diff
@@ -7,23 +7,39 @@
 from arelle.XmlUtil import dateunionValue
 
 LEGALENTITYAXIS_DEFAULT = ''
-_dei_pattern = re.compile(r'^http://xbrl\.((sec\.gov)|(us))/dei/\d{4}-\d{2}-\d{2}')
+_dei_pattern = (
+    re.compile(r'^http://xbrl\.((sec\.gov)|(us))/dei/\d{4}-\d{2}-\d{2}'))
 _CODE_NAME = 'DQC.US.0005'
 _RULE_VERSION = '1.0'
 
 
 def _get_end_of_period(val):
     """
-    Gets the end of period date (dei:DocumentPeriodEndDate).  Returns the fact value or the context end date on the fact, whichever is later,
-    in a tuple with the fact itself and a string representation of the date. The tuple is in a dict keyed off of any LegalEntityAxis members.
-    The end of period string is adjusted for viewing to represent the day as expected.  In other words, dates that end in 24:00 will be put at 00:00 of the expected day.
+    Gets the end of period date (dei:DocumentPeriodEndDate).  Returns the fact
+    value or the context end date on the fact, whichever is later, in a tuple
+    with the fact itself and a string representation of the date. The tuple is
+    in a dict keyed off of any LegalEntityAxis members. The end of period
+    string is adjusted for viewing to represent the day as expected.  In other
+    words, dates that end in 24:00 will be put at 00:00 of the expected day.
 
-    @return A dictionary of tuples containing the fact, found date and a string representation of that date, keyed off of the LegalEntityAxis members in the format {lea_member: (fact, found_date, date_str)}
+    :param val: bal from which to gather end dates
+    :type val: val
+    :rtype: dictionary of tuples
+    :return: A dictionary of tuples containing the fact, found date and a
+    string representation of that date, keyed off of the LegalEntityAxis
+    members in the format {lea_member: (fact, found_date, date_str)}
     """
     results = {}
-    end_of_period_concepts = [c for c in val.modelXbrl.nameConcepts['DocumentPeriodEndDate'] if c.qname.namespaceURI in val.disclosureSystem.standardTaxonomiesDict]
+    end_of_period_concepts = [
+        c for c in val.modelXbrl.nameConcepts['DocumentPeriodEndDate']
+        if c.qname.namespaceURI in val.disclosureSystem.standardTaxonomiesDict
+    ]
+
     if len(end_of_period_concepts) == 1:
-        end_of_period_dict = facts.LegalEntityAxis_facts_by_member(val.modelXbrl.factsByQname[end_of_period_concepts[0].qname])
+        end_of_period_dict = \
```

Comment:
instead of using the \ to break up the line, should put parens around the line and break it up like normal